src/api/public/apidocs/OBS-v2.10.50.yaml

```diff
@@ -443,6 +443,20 @@ paths:
     $ref: 'paths/statistics_most_active_packages.yaml'
   /statistics/most_active_projects:
     $ref: 'paths/statistics_most_active_projects.yaml'
+  /statistics/activity/{project_name}/{package_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/added_timestamp/{project_name}/{package_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/updated_timestamp/{project_name}/{package_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/active_request_creators/{project_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/global_counters:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
+  /statistics/maintenance_statistics/{project_name}:
+    $ref: 'paths/statistics_deprecated_not_documented.yaml'
```

Comment:
Sorry, I didn't realize this before. The endpoints should be displayed alphabetically

Reply:
You're right, I missed that, good catch. Fixed now